spec/micro/micro_spec.lua

```diff
@@ -0,0 +1,50 @@
+local micro = require("spec.micro.micro")
+local snowplow = require("snowplow")
+
+describe("micro", function()
+  before_each(micro.clear_cache)
+
+  it("clears the cache", function()
+    local t = snowplow.new_tracker(micro.get_url())
+    for _ = 1, 20 do
+      local ok, err = t:track_struct_event("category", "action")
+      assert.is_true(ok)
+      assert.is_nil(err)
+    end
+
+    assert.is_equal(20, #micro.get_good_events())
+    micro.clear_cache()
+    assert.is_equal(0, #micro.get_good_events())
+  end)
+
+  it("can get all events from /good with no filter)", function()
+    local t = snowplow.new_tracker(micro.get_url())
+    for _ = 1, 20 do
+      local ok, err = t:track_struct_event("category", "action")
+      assert.is_true(ok)
+      assert.is_nil(err)
+    end
+
+    local data = micro.get_good_events()
```

Comment:
I'm a bit baffled that you didn't have to put a sleep here to wait for Micro in between requests, but it seems to work so that's great!